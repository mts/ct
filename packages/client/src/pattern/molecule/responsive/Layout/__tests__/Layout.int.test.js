import { defaultRender } from './Layout.int.render'

describe('<Layout />', () => {
  describe('Snaphot', () => {
    test('must match defaultRender', () => {
      expect(global.renderToJSON(defaultRender)).toMatchSnapshot()
    })
  })
})
