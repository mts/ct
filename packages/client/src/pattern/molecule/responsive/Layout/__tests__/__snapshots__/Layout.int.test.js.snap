// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Layout /> Snaphot must match defaultRender 1`] = `
<div
  className="layout"
>
  <h1>
    Make This Responsive
  </h1>
  <p>
    While maintaining the heirarchy of importance.
  </p>
  <article
    className="layoutMainStory"
  >
    <img
      alt="Sha Ra Rocking"
      className="layoutMainStoryImage"
      src="https://images-wixmp-ed30a86b8c4ca887773594c2.wixmp.com/f/c4c0230e-617c-4870-b158-1cb768fe3045/d1wzfmg-1c8d8335-f37f-44f6-a463-2ace88617de6.jpg/v1/fill/w_1173,h_681,q_70,strp/she_ra_by_nebezial_d1wzfmg-pre.jpg?token=eyJ0eXAiOiJKV1QiLCJhbGciOiJIUzI1NiJ9.eyJzdWIiOiJ1cm46YXBwOjdlMGQxODg5ODIyNjQzNzNhNWYwZDQxNWVhMGQyNmUwIiwiaXNzIjoidXJuOmFwcDo3ZTBkMTg4OTgyMjY0MzczYTVmMGQ0MTVlYTBkMjZlMCIsIm9iaiI6W1t7ImhlaWdodCI6Ijw9OTI5IiwicGF0aCI6IlwvZlwvYzRjMDIzMGUtNjE3Yy00ODcwLWIxNTgtMWNiNzY4ZmUzMDQ1XC9kMXd6Zm1nLTFjOGQ4MzM1LWYzN2YtNDRmNi1hNDYzLTJhY2U4ODYxN2RlNi5qcGciLCJ3aWR0aCI6Ijw9MTYwMCJ9XV0sImF1ZCI6WyJ1cm46c2VydmljZTppbWFnZS5vcGVyYXRpb25zIl19.Io2zCTwgtYJ2Zl2nNHvyjji31ONmF4uZxz5DYwnwLtg"
    />
    <div
      className="layoutMainStoryIntro"
    >
      <h1>
        Most Important Story
      </h1>
      <p>
        This article has the most visual weight.
      </p>
    </div>
  </article>
  <section
    className="layoutSubStoryList"
  >
    <article
      className="layoutSubStoryListItem"
    >
      <img
        alt=""
        className="layoutSubStoryListItemImage"
        src="http://placebear.com/250/350"
      />
      <div
        className="layoutSubStoryListItemIntro"
      >
        <h2>
          Less Important Story
        </h2>
        <p>
          This story has less visual weight.
        </p>
      </div>
    </article>
    <article
      className="layoutSubStoryListItem"
    >
      <img
        alt=""
        className="layoutSubStoryListItemImage"
        src="http://placekitten.com/250/350"
      />
      <div
        className="layoutSubStoryListItemIntro"
      >
        <h2>
          Less Important Story
        </h2>
        <p>
          This story has less visual weight.
        </p>
      </div>
    </article>
    <article
      className="layoutSubStoryListItem"
    >
      <img
        alt=""
        className="layoutSubStoryListItemImage"
        src="http://placebear.com/250/350"
      />
      <div
        className="layoutSubStoryListItemIntro"
      >
        <h2>
          Less Important Story
        </h2>
        <p>
          This story has less visual weight.
        </p>
      </div>
    </article>
  </section>
</div>
`;
